src/lm_buddy/jobs/evaluation/prometheus.py

```diff
@@ -107,25 +114,27 @@ def run_eval(config: PrometheusJobConfig) -> Path:
     # Instantiate OpenAI client to speak with the vLLM endpoint
     client = OpenAI(base_url=config.prometheus.inference.base_url)
 
-    # load dataset from W&B artifact
     hf_loader = HuggingFaceAssetLoader()
+
+    # Resolve the engine model
+    engine_path = hf_loader.resolve_asset_path(config.prometheus.inference.engine)
```

Comment:
This was an issue left over after the path changes.

if the `engine` field has the path prefixes like `hf://` or `wandb://`, then it has to be resolved before passing to the OpenAI client for completions.